docs/csharp/programming-guide/concepts/expression-trees/index.md

```diff
@@ -104,7 +104,7 @@ Console.WriteLine(factorial);
 // Prints 120.  
 ```  
   
- For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010](http://go.microsoft.com/fwlink/?LinkId=169513), which also applies to later versions of Visual Studio.
+ For more information, see [Expression Trees (C#)](../../../../csharp/programming-guide/concepts/expression-trees/index.md), which also applies to later versions of Visual Studio.
```

Comment:
I meant, your first change looked like this:
```markdown
For more information, see [Generating Dynamic Methods with Expression Trees in Visual Studio 2010](http://go.microsoft.com/fwlink/?LinkId=169513), which also applies to later versions of Visual Studio.
```
So if that blog post contains useful information that we don't have in our docs, it should probably be added as an official topic. Makes sense? So my suggestion was to keep the link to the blog post for now.